Makefile

```diff
@@ -127,7 +127,7 @@ update-service: build-in-docker
 update: build-all
 	docker push $(SERVICE)
 
-update-minimal: build-minimal
+update-minimal:
```

Comment:
why not to call build-minimal? we still need to get binary somehow no?

Reply:
The binary is built as part of the [Dockerfile](https://github.com/openshift/assisted-service/blob/bed493098c712a9bafe627cda1e1ec52e35a65d1/Dockerfile.assisted-service#L22)